### PR TITLE
New package: Microsoft.MediaCreationTool version 10.0.22621.2714

### LINKED ISSUE #141315: [Package Request]: Windows 11 Media Creation Tool
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://www.microsoft.com/software-download/windows11
Publisher: Microsoft
Package Name: Windows 11 Media Creation Tool
Description: create Windows 11 installation media
Package Version: 10.0.22621.2714
Installer URL: https://download.microsoft.com/download/e/c/d/ecd532eb-bed0-465a-9b7a-330066bec3ce/MediaCreationTool_Win11_23H2.exe

Thanks for your kind consideration!

### PR DESCRIPTION
**Pull request by Exorcism0666**
- [X] Resolve: #141315
---

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping Exorcism0666.
>[!warning]
>With the new version of Komac, numerous Pull Requests may contain bugs or unwanted elements. If such issues arise, please tag russellbanks so that these bugs can be addressed in future updates.

>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
Manual Validation ended with:
image

File: manifests/m/Microsoft/MediaCreationTool/10.0.22621.2714/Microsoft.MediaCreationTool.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Microsoft.MediaCreationTool
+PackageVersion: 10.0.22621.2714
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+PackageName: Media Creation Tool
+License: Proprietary
+Copyright: © Microsoft Corporation. Tous droits réservés.
```
Comment (stephengillie) [inline manifests/m/Microsoft/MediaCreationTool/10.0.22621.2714/Microsoft.MediaCreationTool.locale.en-US.yaml:10]:
```suggestion
Copyright: © Microsoft Corporation. All rights reserved.
```

**Review by stephengillie — APPROVED** [review]

**Review by Trenly — APPROVED** [review]

**Review by Trenly — APPROVED** [review]

**Comment by Trenly** [comment]
.

**Comment by RokeJulianLockhart** [comment]
> https://github.com/microsoft/winget-pkgs/pull/141318#event-11974080189

@stephengille and @Trenly, where does this install do? There's no obvious way to invoke this after installing it.

**Comment by RokeJulianLockhart** [comment]
Forgot https://github.com/microsoft/winget-pkgs/discussions/184459#discussioncomment-10996601. Apologies.

**Pull request merged**